Softeq.XToolkit.WhiteLabel.Droid/Internal/ViewModelStore.cs

```diff
@@ -6,10 +6,15 @@
 
 namespace Softeq.XToolkit.WhiteLabel.Droid.Internal
 {
-    internal static class ViewModelStore
+    public static class ViewModelStore
     {
         private const string ViewModelStoreTag = "WL_ViewModelStore";
 
+        public static void SetupFor(FragmentManager fragmentManager)
```

Comment:
what about use `Of()` method below?

Reply:
Method `Of()` and interface `IInstanceStorage` are internal - I assume it was made for a reason. Besides, method `Of()` doesn't really show the intention (to setup ViewModelStore on a FragmentManager before usage), it's just happened to have similar implementation